pkg/mount/mount.go

```diff
@@ -543,6 +548,33 @@ func (m *Mounter) removeSoftlinkAndTarget(link string) error {
 	return nil
 }
 
+// isPathSetImmutable returns true on error in getting path info or if path
+// is immutable .
+func (m *Mounter) isPathSetImmutable(mountpath string) bool {
+	if _, err := os.Stat(mountpath); err != nil {
+		dlog.Errorf("Failed to stat mount path:%v", err)
+		return true
+	}
+	op, err := exec.Command(lsattrBin, "-d", mountpath).CombinedOutput()
+	if err != nil {
+		// Cannot get path status, return true so that immutable bit is not reverted
+		dlog.Errorf("error listing attrs for %v err:%v", mountpath, string(op))
+		return true
+	}
+	attrs := strings.Split(string(op), " ")
+	if len(attrs) < 2 {
+		// Cannot get path status, return true so that immutable bit is not reverted
+		dlog.Errorf("Invalid lsattr output %v", string(op))
+		return true
+	}
+	if strings.Contains(attrs[0], "i") {
+		dlog.Errorf("Mount Path already set to Immutable")
```

Comment:
Should probably just be a warning, this function is just checking the state, it isn't an error for this function.

Reply:
it isn't error. But Not expected. its good leave that log.